mne/viz/_brain/_brain.py

```diff
@@ -2456,6 +2456,31 @@ def set_time_point(self, time_idx):
         self._data['time_idx'] = time_idx
         self._update()
 
+    def set_time(self, time_sec):
+        """Set the time to display (in seconds).
+
+        Parameters
+        ----------
+        time_sec : float
+            The time to show.
+        """
+        if self._times is None:
+            raise ValueError(
+                'Cannot set time when brain has no defined times.')
+        elif min(self._times) <= time_sec <= max(self._times):
+            idx = np.searchsorted(self._times, time_sec)
+            if time_sec in self._times:
+                self.set_time_point(idx)  # requested exact match, pass an int
+            else:
+                step = np.diff(self._times[idx - 1:idx + 1])
+                rel = (time_sec - self._times[idx - 1]) / step
+                print(idx, step, rel)
+                self.set_time_point(idx + rel)
```

Comment:
There is some debugging cruft in here. I think we can simplify with
```Suggestion
            self.set_time_point(np.interp(float(time_sec), self._times, self._times))
``````

Reply:
sorry about the print cruft.  Your suggestion to use `np.interp` is good, but I think it should be `np.interp(float(time_sec), self._times, np.arange(self._n_times))` or so, right?  The way you have it it will be a no-op.

Reply:
Ahh yes, thanks